dockerfiles/init/manifests/che.env

```diff
@@ -455,6 +455,29 @@ CHE_SINGLE_PORT=false
 # Note that this property is needed for backward compatibility and will be removed soon.
 #CHE_PREDEFINED_STACKS_RELOAD__ON__START=false
 
+
+
+########################################################################################
+#####                                                                              #####
+#####                                 MAIL SERVER                                  #####
+#####                                                                              #####
+#
+# CHE uses email to notify admins and users of events in the system such as error
+# reports invitations, or workspace unusual activity.
+#
+#CHE_MAIL_HOST=smtp.example.com
+#CHE_MAIL_HOST_PORT=465
```

Comment:
these lines are commented, i/e it won't be injected, where do you have default values for it? I mean we usually have them in che.properties or hardcoded in code but I don't see any

Reply:
Nowhere . By default, it will fail.

Reply:
so with these changes che will fail to start, right? is that acceptable?

Reply:
I would not fail on tomcat start. I think we can uncomment this default values in future if we would like to do so.